mod/photos.php

```diff
@@ -965,7 +993,6 @@ function photos_content(App $a) {
 		return;
 	}
 
-
 	require_once('include/bbcode.php');
```

Comment:
Let's push that to next PR, okay? Because it occurred 27 times over about nearly the same amount of files.